helm-chart/values.yaml.changelog.md

```diff
@@ -5,6 +5,13 @@ For changes that require manual steps other than changing values, please check o
 Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
+## Upgrading to Renku 0.66.0
+
+* NEW `dataService.imageBuilders` to configure session image builds using Shipwright.
+  This feature is experimental and should not be enabled by RenkuLab administrators.
```

Comment:
it should _not_ be enabled?

Reply:
we list Shipwright as a requirement and the feature as experimental - I think that's fine